hass_nabucasa/__init__.py

```diff
@@ -412,20 +427,53 @@ def _decode_claims(token: str) -> Mapping[str, Any]:
         )
         return decoded
 
-    def async_initialize_subscription_expired_handler(self) -> None:
-        """Initialize the subscription expired handler."""
-        if self._subscription_expired_task is not None:
-            _LOGGER.debug("Subscription expired handler already running")
+    def async_initialize_subscription_reconnection_handler(
+        self,
+        variation: SubscriptionReconnectionVariation,
```

Comment:
This better explains what it is, I think.
```suggestion
        reason: SubscriptionReconnectionReason,
```